caliban_toolbox/settings.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2016-2020 The Van Valen Lab at the California Institute of
+# Technology (Caltech), with support from the Paul Allen Family Foundation,
+# Google, & National Institutes of Health (NIH) under Grant U24CA224309-01.
+# All rights reserved.
+#
+# Licensed under a modified Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.github.com/vanvalenlab/caliban-toolbox/LICENSE
+#
+# The Work provided may be used for non-commercial academic purposes only.
+# For any other use of the Work, including commercial use, please contact:
+# vanvalenlab@gmail.com
+#
+# Neither the name of Caltech nor the names of its contributors may be used
+# to endorse or promote products derived from this software without specific
+# prior written permission.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+from __future__ import absolute_import
+from __future__ import print_function
+from __future__ import division
+
+from decouple import config
+
+X_DIMENSION_LABELS = config('DIMENSION_LABELS', cast=list,
+                            default=['fovs', 'stacks', 'crops', 'slices',
+                                     'rows', 'cols', 'channels'])
+
+Y_DIMENSION_LABELS = config('DIMENSION_LABELS', cast=list,
+                            default=['fovs', 'stacks', 'crops', 'slices',
+                                     'rows', 'cols', 'compartments'])
```

Comment:
How does casting to a list work? Does it expect a comma-separated entry or something?